relnotes/taskpool.feature.md

```diff
@@ -0,0 +1,4 @@
+* `ocean.task.TaskPool`
+
+  A new function `awaitAllTasks()` has been added. This function suspends the
+  current task until all tasks in the pool have finished executing.
```

Comment:
I see what you mean. In my particular case, the current task is not a task from the pool at all. It is simply a task which owns the pool. Any suggestion for how to make this better?